src/Container/ApplicationFactory.php

```diff
@@ -41,23 +39,11 @@
  *
  * You may disable injection of configured routing and the middleware pipeline
  * by enabling the `zend-expressive.programmatic_pipeline` configuration flag.
- *
- * You may opt-in to providing a middleware-based error handler by enabling the
- * `zend-expressive.raise_throwables` configuration flag. If you do this, the
- * application will no longer catch exceptions and invoke Stratigility error
- * middleware; you will instead need to provide custom middleware that provides
- * the try/catch block. You may use Zend\Stratigility\Middleware\ErrorHandler,
- * with our provided Zend\Expressive\Middleware\ErrorResponseGenerator and/or
- * Zend\Expressive\Middleware\WhoopsErrorResponseGenerator classes.
- *
- * Additionally, when raise_throwables is enabled, you will need to provide an
- * innermost middleware to invoke when the queue is exhausted; we suggest
- * Zend\Expressive\Middleware\NotFoundHandler.
  */
 class ApplicationFactory
 {
-    const DISPATCH_MIDDLEWARE = 'EXPRESSIVE_DISPATCH_MIDDLEWARE';
-    const ROUTING_MIDDLEWARE = 'EXPRESSIVE_ROUTING_MIDDLEWARE';
+    const DISPATCH_MIDDLEWARE = Application::DISPATCH_MIDDLEWARE;
+    const ROUTING_MIDDLEWARE = Application::ROUTING_MIDDLEWARE;
```

Comment:
Not technically a BC break; just switches the definitive source for the definitions. I'm keeping these, as anybody who was using configuration-based pipelines would have their code break immediately otherwise.